### PR TITLE
IOS and Desktop

### PR DESCRIPTION
**Pull request by fbertsch**
Adds both of those apps, also changes the behavior when running on a single app (this should not affect prod).

### CODE REVIEW

File: repositories.yaml [hunk 1]
```diff
@@ -123,3 +123,19 @@ firefox-reality:
   dependencies:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
+lockwise-ios:
+  app_id: org.mozilla.ios.Lockbox
+  notification_emails:
+    - frank@mozilla.com
+    - tlong@mozilla.com
```
Comment (Dexterp37) [inline repositories.yaml]: I think this should contain also people from the Lockwise team, shouldn't it?

File: repositories.yaml [hunk 2]
```diff
@@ -123,3 +123,19 @@ firefox-reality:
   dependencies:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
+lockwise-ios:
```
Comment (Dexterp37) [inline repositories.yaml]: Will you be tackling Android as a separate PR? (https://bugzilla.mozilla.org/show_bug.cgi?id=1600613)

File: repositories.yaml (hunk 1 above)
Comment (Dexterp37) [reply repositories.yaml]: Maybe `lockwise-dev@mozilla.com`?

File: repositories.yaml (hunk 2 above)
Comment (fbertsch) [reply repositories.yaml]: Sure, we can add Android while we're at it

File: repositories.yaml (hunk 1 above)
Comment (fbertsch) [reply repositories.yaml]: @travis79 can you confirm who the recipients should be?

File: repositories.yaml [hunk 3]
```diff
@@ -123,3 +123,26 @@ firefox-reality:
...
+  notification_emails:
+    - frank@mozilla.com
+    - chutten@mozilla.com
+  url: 'https://github.com/mozilla/gecko-dev'
+  dependencies:
+    - org.mozilla.components:service-glean
+lockwise-android:
+  app_id: mozilla.Lockbox
+  notification_emails:
+    - frank@mozilla.com
```
Comment (Dexterp37) [inline repositories.yaml]: I think we should add `lockwise-dev@mozilla.com` here as well

File: repositories.yaml (hunk 1 above)
Comment (travis79) [reply repositories.yaml]: lockwise-dev@mozilla.com is good and maybe ddurst@mozilla.com?

File: repositories.yaml (hunk 3 above)
Comment (fbertsch) [reply repositories.yaml]: Cool, added

File: repositories.yaml (hunk 1 above)
Comment (fbertsch) [reply repositories.yaml]: Done and done

File: repositories.yaml [hunk 4]
```diff
@@ -123,3 +123,28 @@ firefox-reality:
...
+lockwise-ios:
+  app_id: org.mozilla.ios.Lockbox
+  notification_emails:
+    - frank@mozilla.com
+    - tlong@mozilla.com
+    - lockwise-dev@mozilla.com
+  url: 'https://github.com/mozilla-lockwise/lockwise-ios'
+  dependencies:
+    - org.mozilla.components:service-glean
+firefox-desktop:
```
Comment (chutten) [inline repositories.yaml]: The fogotype is a "burn after using" prototype. Will attaching it to the "firefox-desktop" name here cause naming collisions down the line when we remove this one after fogotype's end, and want to add it back when we integrate the Glean SDK for real?

Comment (fbertsch) [reply repositories.yaml]: It will indeed cause naming collisions. If this is a prototype let's use a different name. @chutten opinions on what it should be? `fog` seems short and unclear.

Comment (chutten) [reply repositories.yaml]: `fogotype` works for me, as that's the "application" (heavy use of airquotes) that's sending the "prototype" pings with the app id `org.mozilla.fogotype`. Sound good, @fbertsch ?

**Comment by chutten** [comment]
@fbertsch Anything I can help unblock on this?

**Comment by fbertsch** [comment]
@chutten we should be good to go to merge, I was waiting for someone to approve :) @Dexterp37?

**Comment by Dexterp37** [comment]
Redirecting to @mdboom as I'm at the end of my day
